demos/python_demos/im2latex_demo/im2latex_demo.py

```diff
@@ -0,0 +1,230 @@
+#!/usr/bin/env python3
+"""
+ Copyright (c) 2020 Intel Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import logging as log
+import os
+import sys
+from argparse import SUPPRESS, ArgumentParser
+
+import cv2 as cv
+import numpy as np
+from tqdm import tqdm
+
+from openvino.inference_engine import IECore
+
+from utils import END_TOKEN, START_TOKEN, Vocab
+
+def crop(img, target_shape):
+    target_height, target_width = target_shape
+    img_h, img_w = img.shape[0:2]
+    new_w = min(target_width, img_w)
+    new_h = min(target_height, img_h)
+    img = img[:new_h, :new_w, :]
+    return img
+
+
+def resize(img, target_shape):
+    target_height, target_width = target_shape
+    img_h, img_w = img.shape[0:2]
+    scale = min(target_height / img_h, target_width / img_w)
+    img = cv.resize(img, None, fx=scale, fy=scale)
+    return img
+
+
+PREPROCESSING = {
```

Comment:
Why does the demo have several preprocessing types?

Reply:
Two preprocessing types are used for two different datasets as model trained with concrete font size, so if we want to run the model on inputs with bigger font size (e.g. if input is photographed in 12Mpx, while model trained to imitate scans in ~3Mpx) we should first resize the input to make font size like in train set.

Reply:
Please, point to it into documentation

Reply:
Added information into the readme